participant-manager/src/app/page-not-found/page-not-found.component.html

```diff
@@ -1,15 +1,26 @@
+<header>
```

Comment:
Should this have copyright header?

Reply:
Hi @jkrogers18,  we include copyright headers in only SVG images, not in HTML and CSS files.